add error message in lint when docker unavailable (#2298)
demisto/demisto-sdk: CHANGELOG.md

## Unreleased
* Fixed an issue in **validate**, where backward-compatibility failures prevented other validations from running.

> * Added an error message in **lint** to check if the `docker daemon` is running. 

Reviewer: ```suggestion
* Added an error when the docker daemon is unavailable when running **lint**.
```
